examples/js/components/FileInput.jsx

```diff
@@ -29,14 +40,21 @@ class FileInput extends Component {
           />
           <CardText>
             <SafeFileInput
+              disabled={this.state.disabled}
               onChange={::this.onChange}
             />
             <SafeFileInput
               accept={'.csv'}
+              disabled={this.state.disabled}
               shouldReject={::this.shouldReject}
               onChange={::this.onChange}
               onReject={::this.onReject}
             />
+            <RaisedButton
+              label='Toggle Fields'
+              style={{margin: 12}}
```

Comment:
This should be `style={{margin: '12px'}}`. As of react 15.0 (i think), passing numbers for styling is deprecated.